crowdloans/rewards/polkadot.json

```diff
@@ -1474,22 +1474,33 @@
     "parallelBonus": {
       "items": [
         { "formula": "0.05", "logo": "/static/media/para.e1be6567.svg" },
-        { "formula": "0.10" }
+        { "formula": "0.05" },
+        { "formula": "0.25", "logo": "/static/media/para.e1be6567.svg" }
+      ],
+      "tooltip": [
+        { "text": "Parallel Referral Bonus", "value": "5% PARA" },
+        { "text": "Early Bird Bonus", "value": "5% EFI" },
+        { "text": "Parallel Re-Contributors", "value": "25% PARA" }
       ],
-      "tooltip": [{ "text": "Parallel Referral Bonus", "value": "10% PARA" }],
       "detail": [
         { "label": "Parallel Bonus", "formula": "${amount * 4}" },
         {
           "label": "Parallel Referral Bonus",
           "formula": "${referralAvailable ? amount * 4 * 0.05 : 0}"
         },
         {
-          "label": "Parallel Re-Contributors (Early Bird, before 11/23/2021)",
-          "formula": "${amount * 4 * 0.25}"
+          "label": "Parallel Re-Contributors",
+          "formula": "${amount * 4 * 0.25}",
+          "tooltip": [
+            "Early Bird, before 11/23/2021"
```

Comment:
"Early bird, this applies for users who also contributed in batch 1 before 11/23/2021."